config/vufind/config.ini

```diff
@@ -163,6 +163,14 @@ allowSavedSearches = true
 ; still require Javascript; this simply improves compatibility for certain
 ; features (such as display of hierarchies).
 nonJavascriptSupportEnabled = false
+; Only display x number of subjects in the bib display on the full record by default.
```

Comment:
I think it would make sense to move both of these new settings down to the [Record] section, since they both apply to record display. I'd also rename them for greater specificity (see comments on config method names below).